packages/cli/src/__tests__/ceramic-daemon.test.ts

```diff
@@ -153,23 +153,27 @@ describe('Ceramic interop: core <> http-client', () => {
 
   it('gets anchor commit updates', async () => {
     const doc1 = await TileDocument.create(core, { test: 123 })
+    await TestUtils.hasAcceptedAnchorRequest(core, doc1.tip)
```

Comment:
I really don't like seeing these scattered throughout all of our tests.  Why is this necessary?

Reply:
And, it's gone.